grid-demo/src/app/angular/helpers/datagrid-active-cell.helpers.ts

```diff
@@ -0,0 +1,36 @@
+import { ClrDatagrid } from '@clr/angular';
+import { Observable } from 'rxjs';
+import { publish, refCount } from 'rxjs/operators';
+
+import { getDatagridKeyNavigationController } from './datagrid-private-member.helpers';
+
+const activeCellObservableCache = new WeakMap<ClrDatagrid, Observable<HTMLElement>>();
+
+export function fromActiveCell(datagrid: ClrDatagrid) {
+  return activeCellObservableCache.get(datagrid) || createAndCacheActiveCellObservable(datagrid);
+}
+
+function createAndCacheActiveCellObservable(datagrid: ClrDatagrid) {
+  const activeCellObservable = createActiveCellObservable(datagrid).pipe(publish(), refCount());
```

Comment:
nit: publish+refCount seems to be deprecated. Looking at the docs, something like the following should be the suggested update:
```
createActiveCellObservable(datagrid).pipe(
  share({
    resetOnError: false,
    resetOnComplete: false,
    resetOnRefCountZero: false
  })
  ...
  ```

Reply:
Thanks!

Reply:
We aren't using RxJS 7.

Reply:
I went ahead and updated to RxJS 7.

Reply:
You were right. Didn't notice we're behind. This was more of a question for my own learning. Thanks!